test/tools/yulopti.cpp

```diff
@@ -59,6 +60,19 @@ using namespace solidity::yul;
 
 namespace po = boost::program_options;
 
+namespace
+{
+
+string stringToLower(string const& _input)
```

Comment:
Use a boost function?

Reply:
[`boost::iequals()`](https://www.boost.org/doc/libs/1_75_0/doc/html/boost/algorithm/iequals.html) was the first thing I tried but I was always getting `false` from it so I decided it's not worth the time figuring it out. Now that you mention it, I realized that I should have been using [`is_iless()`](https://www.boost.org/doc/libs/1_75_0/doc/html/boost/algorithm/is_iless.html) :) I'll change that.

Reply:
I mean a function to compute `tolower` of a string

Reply:
True, I shuld have used `boost::algorithm::to_lower()`.

But anyway, the only thing I needed `stringToLower()` for was the case-insensitive comparison. `is_iless()` works so I'm using that now.